lib/include/openamp/rpmsg.h

```diff
@@ -159,7 +160,11 @@ int rpmsg_send_offchannel_raw(struct rpmsg_endpoint *ept, uint32_t src,
  * one becomes available, or a timeout of 15 seconds elapses. When the latter
  * happens, -ERESTARTSYS is returned.
  *
- * Returns number of bytes it has sent or negative error value on failure.
+ * @param ept	The rpmsg endpoint
+ * @param data	Payload of the message
+ * @param len	Length of the payload
+ *
+ * @return Number of bytes it has sent or negative error value on failure.
  */
 static inline int rpmsg_send(struct rpmsg_endpoint *ept, const void *data,
```

Comment:
seems that doxygen does not generate the documentation for static function. I don't know if there is a way to force it.... do you have a simple solution?

An alternative (which would get advantage to improve the support of RUST client) would be an implementation similar to 
this proposal:
https://github.com/OpenAMP/libmetal/pull/159/commits/ef0a476ae43aa7f9cf7443c88e1163fee1def454

For this release we can just enter an issue to fix that later.

Reply:
Agree, Let treat "static" declaration issue in a next step and get this one for this release.